### PR TITLE
Refactored markup and styling  (#1zw388m)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

File: src/components/Logo.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+})
+</script>
+
+<style scoped>
+
+figure {
+  text-align: center;
+}
+
+figure > img {
```
Comment (dt2patel) [inline src/components/Logo.vue]: The width of the logo should not be defined in the component but during usage of the component on the login and home page

File: src/views/Home.vue [hunk 2]
```diff
@@ -2,98 +2,97 @@
...
-            <ion-label>Settings</ion-label>
+        <nav>
+          <section class="external">
+            <ion-card href="https://preorder.hotwax.io/login">
+              <img src="../assets/images/PreOrder.svg" />
+              <ion-card-header>
+                <ion-card-title>Pre Order Management</ion-card-title>
+              </ion-card-header>
+            </ion-card>
+            <ion-card href="">
```
Comment (dt2patel) [inline src/views/Home.vue]: add route to https://threshold-management.hotwax.io/

**Review by dt2patel — CHANGES_REQUESTED** [review]
changes around logo component

File: src/views/Home.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/Home.vue]: Done

File: src/components/Logo.vue (hunk 1 above)
Comment (azkyakhan) [reply src/components/Logo.vue]: Width of image inside figure is not getting over rided on home page. I tried using selectors but its not working.

Comment (dt2patel) [reply src/components/Logo.vue]: Try setting width of image in figure to 100% here and then setting width of logo component in home page

Comment (azkyakhan) [reply src/components/Logo.vue]: I tried this but its not working then I tried to set width and margin of figure component on Home page :

localhost_8100_home (1)

File: src/views/Home.vue [hunk 3]
```diff
@@ -136,6 +136,10 @@ export default defineComponent({
 </script>
 
 <style scoped>
+figure {
```
Comment (dt2patel) [inline src/views/Home.vue]: indentation

**Review by dt2patel — CHANGES_REQUESTED** [review]

**Review by dt2patel — APPROVED** [review]

File: src/views/Home.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/Home.vue]: Done sir

**Pull request merged**